module_3/Dao/src/main/java/com/epam/ems/dto/fields/Constants.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.ems.dto.fields;
+
+public class Constants {
+    public static String USER_ID="id";
+    public static String CERTIFICATES="certificates";
+    public static String OVERAGE_ORDER_PRICE="overageOrderPrice";
```

Comment:
Why aren't these finals?
+ snippet is not clean: equal sign should have whitespaces